Sources/Helpers/Factories/CellItemFactory.swift

```diff
@@ -1,19 +1,25 @@
 //
-//  CellItemFactory.swift
-//
 //  Copyright © 2019 Rosberry. All rights reserved.
 //
 
 import UIKit
 
 public protocol CellItemFactory {
```

Comment:
Should it be called this way?
```suggestion
public protocol CellItemsFactory {
```

Reply:
This will be merged in following PR